### ISSUE TITLE
Video resources

### ISSUE THREAD

**Issue by Pasqualle** [typebytes/ngx-template-streams#25]
I have found the following video resources quite interesting, maybe they should be included in the readme.

How to use it (ngAir211): https://www.youtube.com/watch?v=tbr03N3WA2Q
How it works (NG-BE 2019): https://www.youtube.com/watch?v=RcKB6NCrDPA

**Comment by d3lm** [comment]
Oh very good point! Thanks for suggesting it. I ll add it to the readme in a new section 🙏 

How is it going for you? Are you using this library? Any problems you encounter?

**Comment by Pasqualle** [comment]
No, I am not using the library, I am not even an Angular developer. I am just learning about Angular whenever I have some time.
This library seems like a very logical step for Angular. I hope it will be part of it one day.
Similarly as observables should be part of ecmascript. I wonder which one will happen first.

**Comment by d3lm** [comment]
I 100% agree. I would love to see this part of the core, same goes for Observables and JS.
